cli/install/template.go

```diff
@@ -572,10 +569,6 @@ spec:
           httpGet:
             path: /api/health
             port: 3000
-          initialDelaySeconds: 30
-          timeoutSeconds: 30
-          failureThreshold: 10
-          periodSeconds: 10
```

Comment:
Same question here as above -- won't this only give the grafana pod 20 seconds to start? Maybe we can set `failureThreshold: 7` on this probe as well?

Reply:
Unrelated to your change, but I realized the the grafana pod's liveliness probe does not include an initial delay, whereas the prometheus pod's liveliness probe includes `initialDelaySeconds: 30`. Would you mind updating the grafana pod's liveliness probe to match? I actually wonder if the fact that we haven't previously had an initial delay for the grafana pod's liveliness probe could be the cause of #1747.

Reply:
Good idea around setting `initialDelaySeconds: 30` for Grafana liveness, PR updated.

We should keep this technique in mind if we encounter liveness failures/restarts at startup time, for control or data plane. There's not much harm in increasing `livenessProbe.initialDelaySeconds`.